pkg/test-infra/tidb/recommend.go

```diff
@@ -129,6 +130,54 @@ func RecommendedTiDBCluster(ns, name string) *TiDBClusterRecommendation {
 				},
 			},
 		},
+		TidbMonitor: &v1alpha1.TidbMonitor{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: ns,
+				Labels: map[string]string{
+					"app":      "tipocket-tidbmonitor",
+					"instance": name,
+				},
+			},
+			Spec: v1alpha1.TidbMonitorSpec{
+				Clusters: []v1alpha1.TidbClusterRef{
+					{
+						Namespace: ns,
+						Name:      name,
+					},
+				},
+				Persistent: false,
+				Prometheus: v1alpha1.PrometheusSpec{
+					MonitorContainer: v1alpha1.MonitorContainer{
+						BaseImage: "prom/prometheus",
+						Version:   "v2.11.1",
+					},
+					LogLevel: "info",
+				},
+				Grafana: &v1alpha1.GrafanaSpec{
+					Service: v1alpha1.ServiceSpec{
+						Type: corev1.ServiceType(fixture.Context.TiDBMonitorSvcType),
+					},
+					MonitorContainer: v1alpha1.MonitorContainer{
+						BaseImage: "grafana/grafana",
+						Version:   "6.0.1",
+					},
+				},
+				Initializer: v1alpha1.InitializerSpec{
+					MonitorContainer: v1alpha1.MonitorContainer{
+						BaseImage: "pingcap/tidb-monitor-initializer",
+						Version:   "v3.0.5",
+					},
+				},
+				Reloader: v1alpha1.ReloaderSpec{
+					MonitorContainer: v1alpha1.MonitorContainer{
+						BaseImage: "pingcap/tidb-monitor-reloader",
+						Version:   "v1.0.1",
+					},
+				},
+				ImagePullPolicy: corev1.PullAlways,
```

Comment:
Well. Is it better to use `IfnotPresent`?

Reply:
You are right, `IfNotPresent` is better, I will update it.